.github/ISSUE_TEMPLATE

```diff
@@ -4,8 +4,6 @@ Environment
 -----------
 * Python version: (python -V)
 * deepgnn-ge Version: (python -m pip show deepgnn-ge)
```

Comment:
Just deepgnn now?

Reply:
Just deepgnn means we need to release a new package. It might be useful later, but in this PR we want only to stop publishing -tf/-torch packages.

Reply:
I see what you are saying but if we do them one at a time, users will have to update their pipelines and local installs twice deepgnn-torch -> deepgnn-ge -> deepgnn

Reply:
Only if we rename package to deepgnn as you are suggesting. We are not changing existing deepgnn-torch packages, but stop publishing new versions.